### ISSUE TITLE
Project Section is not accessible using the keyboard: A11y_.NET 9 template_.NET 9 template_Project_Keyboard.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#26815]
### GitHub Tags
#A11yTCS; #A11ySev1; #Win32; #WCAG2.1.1; #SH_.NET9template_MAC_Dec2024; #DesktopApp; #.NET Core; #; #A11yMAS; #Keyboard; #MAC;#AILimited;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
MAC Version 15.2

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till projects section.
4. Observe that Project Section is not accessible using the keyboard keys:

**Similar issue is also observed for tags.**

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till project section, and press enter on any project, Project screen will open,
4. Tab till tags.
5. Observe that tags are not accessible using the keyboard keys.

**Similar issue is also observed for Projects options.**

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till Hamburger menu and press enter, Projects options will screen will open.
4. Observe that Projects options are not accessible using the keyboard keys.

**Similar issue is also observed for Dark/Light theme button**

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till Hamburger menu and press enter.
4. Observe that Dark/light theme button is not accessible using the keyboard keys.

### Actual Result:
Project Section is not accessible using the keyboard keys.

### Expected Result:
Project Section should be accessible using the keyboard keys.

### User Impact:
keyboard users will not be able to access the controls and will not be able to access the other functionality.

### Attachment:
tags are not accessible using the keyboard.

https://github.com/user-attachments/assets/c33e95f6-6552-49bb-8e46-e323f54f845a

Project options

https://github.com/user-attachments/assets/24c225d4-33e7-4843-a7f9-446c3f7cf45a

Project section

https://github.com/user-attachments/assets/ae0ea85d-8bf7-4472-b12b-9d6151488d6b

 Dark/light theme button is not accessible using the keyboard.

https://github.com/user-attachments/assets/3e1cd097-8d76-4ffb-bd30-3ff7e81a6d18

**Comment by InduPriya1805** [comment]
Rev:inse
